Makefile

```diff
@@ -40,6 +40,14 @@ CLI_PLATFORMS ?= linux-amd64 linux-arm linux-arm64 darwin-amd64 windows-amd64 li
 CONTAINER_PLATFORMS ?= linux-amd64 linux-ppc64le linux-arm linux-arm64
 MANIFEST_PLATFORMS ?= amd64 ppc64le arm arm64
 
+#kubebuilder automatically runs "make" when a resource or controller is created
+all: generate-all
```

Comment:
This is just creating an alias for `generate-all`.
You can undo this change and keep `generate-all` as the first target, if that is you wanted.
You can also undo the change on line #92